libraries/imgui-opencv-poco/conanfile.py

```diff
@@ -0,0 +1,25 @@
+from conans import ConanFile, CMake
+
+
+class ImguiOpencvDemo(ConanFile):
```

Comment:
There are both `txt` and `py` files for "historical" reasons because this code was used for demoing and I wanted to show how to transition from one `conanfile.txt` to `conanfile.py`, we can remove one of them.

Reply:
Removed conanfile.txt as suggested by @czoido